metaflow/plugins/kfp/kfp_decorator.py

```diff
@@ -84,6 +84,11 @@ def step_init(self, flow, graph, step, decos, environment, datastore, logger):
         self.datastore = datastore
         self.logger = logger
 
+        # Add env vars from the optional @environment decorator.
```

Comment:
For reference, step function is also handling environment decorator ad-hoc: https://github.com/Netflix/metaflow/blob/master/metaflow/plugins/aws/step_functions/step_functions.py#L377-L382

Reply:
Nice catch! This comment in the PR is a good comment to add to the code.